src/Microsoft.DotNet.Interactive/Connection/StdIoKernelConnector.cs

```diff
@@ -14,15 +14,27 @@
 using Microsoft.DotNet.Interactive.Events;
 using Microsoft.DotNet.Interactive.Utility;
 
-namespace Microsoft.DotNet.Interactive.Connection
+namespace Microsoft.DotNet.Interactive.Connection;
+
+public class StdIoKernelConnector : IKernelConnector, IDisposable
 {
-   public class StdIoKernelConnector : IKernelConnector
-    {
-        public string[] Command { get; }
+    private MultiplexingKernelCommandAndEventReceiver? _receiver;
+    private KernelCommandAndEventTextStreamSender? _sender;
+    private Process? _process;
+
+    public string[] Command { get; }
```

Comment:
This is a somewhat confusing property name. I had to look at how it's used to understand what it meant, since the word "command" means something very different in the context of .NET Interactive's APIs.